packages/wagtail/modules/services/Wagtail/PagesAPI/index.ts

```
@@ -0,0 +1,25 @@
+  static getPageByPath(path: string, extraParams: any = {}): Promise<IPage> {
+    return this.axios.get(`${this.baseURL}/${this.namespace}/path/`, {
+      params: {
+        fields: '*',
+        html_path: path,
+        ...extraParams,
+      },
+    })
+  }
+
+  static getPage(id: number, type: string): Promise<IPage> {
+    return this.axios.get(`${this.baseURL}/${this.namespace}/${id}/`, {
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)